modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberPopupTest.ts

```diff
@@ -0,0 +1,77 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
+import { SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+const openThrobber = async (editor: Editor, delay?: number): Promise<void> => {
+  editor.setProgressState(true, delay);
+  await new Promise((resolve) => setTimeout(resolve, delay));
+  await UiFinder.pWaitForVisible('throbber to open', SugarBody.body(), '.tox-throbber');
+}
+
+describe('browser.tinymce.themes.silver.throbber.ThrobberPopupTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    theme: 'silver',
+    base_url: '/project/tinymce/js/tinymce',
+    setup: (ed: Editor) => {
+      ed.ui.registry.addMenuItem('test-item', {
+        icon: 'nope',
+        text: 'do the thing',
+        onAction: Fun.die('please do not click this')
+      });
+      ed.ui.registry.addContextMenu('test', {
+        update: Fun.constant('test-item')
+      });
+    },
+    contextmenu: 'test'
+  }, [ Theme ]);
+
+  it('cancels all open notifications when it opens', async () => {
+    const ed = hook.editor();
```

Comment:
I think we're doing `editor` everywhere else 

```suggestion
    const editor = hook.editor();
```